concent_api/core/message_handlers.py

```diff
@@ -464,13 +465,14 @@ def handle_send_force_subtask_results(client_message: message.concents.ForceSubt
 
     current_time = get_current_utc_timestamp()
 
-    if Subtask.objects.filter(  # pylint: disable=no-member
-        subtask_id=task_to_compute.compute_task_def['subtask_id'],
-        state=Subtask.SubtaskState.FORCING_ACCEPTANCE.name,  # pylint: disable=no-member
-    ).exists():
-        return message.concents.ServiceRefused(
-            reason=message.concents.ServiceRefused.REASON.DuplicateRequest,
-        )
+    try:
```

Comment:
same here - why?

Reply:
It is to reduce queries to database. I only one ask database for subtask, and then I check it's state- but I check local variable, not database. This is not perfect solution, it will be changed in https://github.com/golemfactory/concent/pull/778 anyway.

Reply:
here neither